src/audio/volume/volume.c

```diff
@@ -207,6 +208,31 @@ static void vol_sync_host(struct comp_dev *dev, unsigned int num_channels)
 	}
 }
 
+/**
+ * \brief Calculate linear ramp function
+ */
+static int32_t volume_linear_ramp(struct comp_dev *dev, int32_t ramp_time, int channel)
+{
+	struct vol_data *cd = comp_get_drvdata(dev);
+
+	return cd->rvolume[channel] + ramp_time * cd->ramp_coef[channel];
+}
+
+/**
+ * \brief Calculate windows fade ramp function
```

Comment:
what format is ramp time? please put in docstring

Reply:
Done